shared/test/unit/mocks/mock_navigator_moz_settings.js

```diff
@@ -98,6 +118,11 @@
             target: request
           });
         }
+        if ('success' in request.listeners) {
+          request.listeners.success.forEach(function (func) {
+            func();
```

Comment:
wondering if we should have an event-like object as parameter... but let's not add it now if we don't need it in tests, we should refactor some things in this file anyway, we can see lots of similar lines :/